src/components/StreamerView/StreamerViewContent.tsx

```diff
@@ -0,0 +1,53 @@
+import { useConnectionState, useLocalParticipant } from '@livekit/components-react'
+import { ConnectionState, Track } from 'livekit-client'
+import { useTranslation } from '../../modules/translation'
+import { EmptyStreamState } from '../LiveKitEnhancements/EmptyStreamState'
+import { LiveStreamCounter } from '../LiveStreamCounter/LiveStreamCounter'
+import { ParticipantGrid } from '../ParticipantGrid/ParticipantGrid'
+import { ContentWrapper } from './StreamerViewContent.styled'
+
+export function StreamerViewContent() {
+  const { t } = useTranslation()
+  const { localParticipant } = useLocalParticipant()
+  const connectionState = useConnectionState()
+
+  // Check if disconnected
+  const isDisconnected = connectionState === ConnectionState.Disconnected
+
+  // Check if there are any active video tracks (camera or screen share) from the LOCAL participant
+  const hasLocalCamera = Array.from(localParticipant?.videoTrackPublications.values() || []).some(
+    pub => pub.source === Track.Source.Camera && pub.track && !pub.isMuted
+  )
+  const hasLocalScreenShare = Array.from(localParticipant?.videoTrackPublications.values() || []).some(
+    pub => pub.source === Track.Source.ScreenShare && pub.track && !pub.isMuted
+  )
```

Comment:
Same here about the hooks. I don't want to make too much noice, can you check de usage of hooks in other files